### PR TITLE
fix for list tag

### PR DESCRIPTION
**Pull request by summerdevs**

### CODE REVIEW

File: secure/m=create/index.html [hunk 1]
```diff
@@ -479,20 +473,27 @@
...
           </li>
 
-                      <li class="top"><a id="community" class="tl" href="./../../404.html"><span class="ts">Community</span></a>
-              <ul>
-                <li><a href="./../../services/m=hiscore/hiscores.html" class="fly"><span>World 1 Hiscores</span></a></li>
-                <li><a href="./../../services/m=hiscore/hiscores.html?world=2" class="fly"><span>World 2 Hiscores</span></a></li>
-                <li><a href="./../../404.html" class="fly"><span>Contributors</span></a></li>
-              </ul>
-            </li>
+          <li class="top"><a id="community" class="tl" href="./../../404.html"><span
```
Comment (dginovker) [inline secure/m=create/index.html:481]: Why did you change this back to add 404s? It's inconsistent with the rest of the website

Comment (summerdevs) [reply secure/m=create/index.html:481]: I didn't change any of it back. I literally copied and pasted the last news page and edited it to have the new content instead of the old.

Comment (summerdevs) [reply secure/m=create/index.html:481]: I assumed the last news page had the correct format since I didn't make it and it was on the site

Comment (dginovker) [reply secure/m=create/index.html:481]: Ahh, darn. This one is the correct one, it looks like the old news pages didn't get updated properly. The community dropdown should only show HS 1, HS 2 and contributors
